FastSimulation/SimplifiedGeometryPropagator/python/TrackerMaterial_cfi.py

```diff
@@ -0,0 +1,353 @@
+import FWCore.ParameterSet.Config as cms
+
+# Do not change the order of the interaction models unless you know what you are doing.
+# Not used at the moment: "muonBremsstrahlung", "nuclearInteractionFTF"
+#_trackerMaterialInteractionModels = cms.untracked.vstring("trackerSimHits")
+_trackerMaterialInteractionModels = cms.untracked.vstring("pairProduction", "nuclearInteraction", "bremsstrahlung", "energyLoss", "multipleScattering", "trackerSimHits")
+
+# Material effects to be simulated in the tracker material and associated cuts 
+TrackerMaterialBlock = cms.PSet(
```

Comment:
Because this is a `cms.PSet` instead of a `cms.untracked.PSet` the entire description of the geometry is stored in the resultant ROOT file as part of the provenance. I do agree that what geometry is used is something we should track, however I'm not convinced that using the configuration language as the way to store the geometry is the best way forward.

Reply:
Is this maybe related to the comment I've just made above (about the ESProducer)?